src/dynamicsSimulation.cpp

```diff
@@ -473,10 +474,15 @@ void DynamicsSimulation::iniWalkerPosition()
     //Todo: poner esto bien sin el caso de hexapacking
     else if(voxels_list.size() > 0 or params.custom_sampling_area){
         walker.setRandomInitialPosition(params.min_sampling_area,params.max_sampling_area);
-        if(params.computeVolume){
```

Comment:
This shouldn't be removed, you could add another case for the multiple diffusivities, but, for meshes, is important to avoid doing extra computations